lib/mobility-core/src/Kernel/External/Ticket/Kapture/Types.hs

```diff
@@ -0,0 +1,99 @@
+{-
+  Copyright 2022-23, Juspay India Pvt Ltd
+
+  This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+  as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program is
+
+  distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+
+  FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero
+
+  General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+
+module Kernel.External.Ticket.Kapture.Types where
+
+import Data.Aeson
+import Kernel.Prelude
+import Kernel.Types.Common (Money)
+
+data CreateTicketReq = CreateTicketReq
+  { title :: Text,
+    ticket_details :: Text,
+    customer_id :: Text,
+    customer_name :: Maybe Text,
+    phone :: Maybe Text,
+    issue_details :: IssueDetails,
+    ride_details :: Maybe RideInfo,
+    classification :: Text
+  }
+  deriving stock (Show, Eq, Generic)
+  deriving anyclass (FromJSON, ToJSON, ToSchema)
+
+data IssueDetails = IssueDetails
+  { issue_description :: Text,
+    issue_id :: Maybe Text,
+    media_files :: Maybe [Text],
+    sub_category :: Maybe Text,
+    category :: Text
+  }
+  deriving stock (Show, Eq, Generic)
+  deriving anyclass (FromJSON, ToJSON, ToSchema)
+
+data RideInfo = RideInfo
+  { rideshortid :: Maybe Text,
+    customername :: Maybe Text,
+    customerphoneno :: Maybe Text,
```

Comment:
Use Readable names.
Use Camel Case and if needed convert in FromJson toJson instance.